### PR TITLE
Portfolio: Mobile version

### PR DESCRIPTION
**Pull request by jssol**
# Add the contact and the projects sections

In this PR:

- Added social media contact links
- Added four templates for flagships projects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,16 +42,164 @@
...
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <section class="technologies">
+              <p class="technology">css</p>
+              <p class="technology">html</p>
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
```
Comment (HENRYKC24) [inline index.html:144]: - [ ] Good job on your project. You are really doing great. However, I think the above `section` element and `p` tag can be replaced with `ul` and `li` elements respectively. That is a list of technologies after all. What do you think:thinking: ?

File: index.html [hunk 2]
```diff
@@ -20,16 +42,164 @@
...
```
Comment (HENRYKC24) [inline index.html:162]: - [ ] Please, refer to the above comment and do this for all cases.

File: index.html [hunk 3]
```diff
@@ -20,16 +42,164 @@
...
+            <button type="button" class="button">See Project</button>     
+          </article>
+        </section>
+      </section>
```
Comment (HENRYKC24) [inline index.html:202]: - [ ] I think you forgot to add the 'About  myself' section here.

File: index.html [hunk 4]
```diff
@@ -20,16 +42,164 @@
...
+          </article>
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
```
Comment (HENRYKC24) [inline index.html:173]: - [ ] In the Figma template, the project images, lines demarcating different projects in the portfolio are all center aligned. Could you, please, adjust your design to look very identical to that of Figma's? I know you have all it takes to do this. Let's try to do this:muscle:
## Figma Design
image
## Your Design
image

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED:recycle:
Change Requested
## Hi @jssol:wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (jssol) [reply index.html:202]: In this template, there is no 'About me' section, there are only the social media links sections repeated twice, and I thought the first instance of them would serve as the 'About me' section, tell me what you think!

Comment (HENRYKC24) [reply index.html:202]: Please, can I have the link to your project requirements again? I want to show you something about it. I have been searching for it but due to the fact that I have so many reviews performed, I can't really say which one is yours.

Comment (jssol) [reply index.html:202]: ## Links to the project requirements and the portfolio Figma template:

- Here's the link to the requirements of the project: [Portfolio projects requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).
- Here's the link to the Figma template: [Figma template link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10).

Comment (jssol) [reply index.html:203]: >  I think you forgot to add the 'About myself' section here.

_In this template, there is no 'About me' section, there are only the social media links sections repeated twice, and I thought the first instance of them would serve as the 'About me' section, tell me what you think!_

**I believe the 'About me' section is merged directly with the headline section.**

Comment (HENRYKC24) [reply index.html:203]: image
This image means you needed to add the 'About Me' section. 
**pulled from the link above**

Comment (jssol) [reply index.html:203]: I chose Template 4, the one with the dark blue background and a backdrop filter

File: index.html (hunk 4 above)
Comment (jssol) [reply index.html:174]: This one is already fixed, you can check in the live preview.

Comment (jssol) [reply index.html:174]: Live Preview: [https://jssol.github.io/portfolio](https://jssol.github.io/portfolio)

File: index.html (hunk 3 above)
Comment (jssol) [reply index.html:203]: Should I add this section, even though it is not on my Figma design?

Comment (HENRYKC24) [reply index.html:203]: I just checked. Probably, you really do not need to add the section. I just checked a friend's portfolio with your design and he has no 'About Myself' section.
Go ahead and resubmit a pull request and I believe the subsequent reviewer will approve your project if you have affected other changes.
Good luck.

**Review by ReshyResh — APPROVED** [review]
# Hi @jssol , 
### You did a great job addressing the issues pointed out by the previous reviewer!
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**